model/src/main/java/io/syndesis/integration/support/Lists.java

```diff
@@ -28,7 +28,7 @@
      */
     public static <T> List<T> notNullList(List<T> list) {
```

Comment:
Can we inline the whole class ? Its seems to be used only in the ChoiceHandler ....